.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: lint
+
+on:
+  - push
+
+jobs:
```

Comment:
Not a blocker for this PR (this is already a _huge_ step forward thanks!) but we also need to lint our JS code according to Magento's ESLint standards. So having a second job here to do that will help catch all the linter issues before we submit a new version ([here's the line in the current script](https://github.com/degica/komoju-magento/blob/1ed62b07382171af79bba15af5199fb1cde33ea1/bin/lint-check#L12) that checks the JS)

Reply:
I went ahead and just added the jslint since it's pretty simple 😄